src/inject/dynamic-theme/uuid.ts

```diff
@@ -0,0 +1,15 @@
+// `hex` variable contain all available hex values into array.
+function hexify(number: Number) {
+    return number < 16 ? "0" + number.toString(16) : number.toString(16);
+}
+// Secure & Fast UUID generator implementation of https://tools.ietf.org/html/rfc4122 version 4.
+export function generateUUID() {
```

Comment:
I've found out that this is invalid UUID. You can simply copy the last snippet from here https://stackoverflow.com/a/2117523/4137472

And another radical solution: we don't really need UUID, you can rename it to UID, simply generate 4 32bit numbers, convert them `toString(16)` and compose. That will have event less chance of collision than UUID and will not have weird dashes.

Also you can remove comments.